### PR TITLE
ENH: Add CI.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add CI:
- Add `*.yml` configuration files for CI.
- Add the `setup.py` Python setup file.
- Add Python wrapping files.
- Add the `CTestConfig.cmake` file to make the build results be sent to
  the **Insight** project in **open.cdash.org**.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@luisibanez

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+setup(
+    name='itk-stlmeshio',
+    version='1.0.0',
+    author='Luis Ibáñez',
+    author_email='community@itk.org',
+    packages=['itk'],
+    package_dir={'itk': 'itk'},
+    download_url=r'https://github.com/InsightSoftwareConsortium/ITKSTLMeshIO',
+    description=r'MeshIO class to read and write ITK meshes into STL files.',
+    long_description='itk-stlmeshio provides a clas to read and write ITK '
```
Comment (thewtex) [inline setup.py]: class

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                     'meshes into STL files.\n'
+                     'Please refer to:\n'
+                     'Ibáñez L. "STL file format MeshIO class for ITK", '
```
Comment (thewtex) [inline setup.py]: \n

File: setup.py (hunk 1 above)
Comment (jhlegarreta) [reply setup.py]: Thanks @thewtex . Done in  5effb06.

File: setup.py (hunk 2 above)
Comment (jhlegarreta) [reply setup.py]: In reality, the `\n` is not producing the expected new line if one has a look at the PyPI documentation... So I've just left the ones that have been used throughout the rest of the remotes. In this case, the name of the IJ article and the rest of the publication information are not separated by a "new line" commonly in the rest of the remotes.

**Pull request merged**